Update community-plugin-marketplace.md (#747)
mattermost/mattermost-developer-documentation: site/content/extend/plugins/community-plugin-marketplace.md

**Technical requirements (checked by developers of the Toolkit team or the Integrations team)**

1. The plugin works for 60k concurrent connections and in a High Availability deployment. **Note:** There are currently no publicly-available tools to verify these properties. As such, they are checked during code review by a developer.
2. The plugin logs important events on appropriate log levels to allow System Admins to troubleshoot issues.

**Security requirements (checked by a member of the Security team)**

1. Security reviews do not reveal any exploitable vulnerabilities in the plugin.
2. The plugin provides an email address or a username on the [Community Server](https://community.mattermost.com) used to report vulnerabilities in the future.

**Functional requirements (checked by a QA tester)**

1. The plugin must set a `min_server_version` in the manifest.
2. The plugin must work on all Mattermost versions greater than or equal to the `min_server_version`.

**Documentation requirements (checked by a Technical Writer)**

1. The plugin must include a `README.md` file with:
    * Requirements
    * Installation steps
    * Configuration steps
    * Usage

>     * Link or email address for help/support

Reviewer: With these items in place does that mean item 4. at "Product requirements" is not longer needed? Should we combine those?

Author: Good catch. I'll combine them.

>     * Link or email address for help/support

Reviewer: Can the support link just be the github issues page? If so, what is the difference to item 3 on "Product requirements"?

> 2. The `plugin.json` file should include formatting consistent with the System Console UI. For assistance with this, you can visit the [Documentation channel](https://community.mattermost.com/core/channels/documentation) for assistance.

Reviewer 2: ```suggestion
2. The `plugin.json` file should include formatting consistent with the System Console UI. You can join the [Documentation channel](https://community.mattermost.com/core/channels/documentation) for assistance.
```

> 2. The `plugin.json` file should include formatting consistent with the System Console UI. For assistance with this, you can visit the [Documentation channel](https://community.mattermost.com/core/channels/documentation) for assistance.

Reviewer: Are there any docs about the style guidelines that we can link?

**Product requirements (checked by a Product Manager)**

1. The plugin is published under an [Open Source license](https://opensource.org/licenses/alphabetical).
2. The source code is available in a public Git repository.
3. There is a public issue or bug tracker for the plugin, which is linked in the plugin documentation and linked via `support_url` in the manifest.
4. The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features, and a development guide. This is typically a `README` file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [`README` of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
5. For the current release and future releases, a changelog must be published. The link to the current release notes has to be recorded in the `release_notes_url` property of the `plugin.json` manifest. For example GitHub releases can be used to publish the changelog.
6. The plugin has to be out of Beta and be released with at least v1.0.0.

> 7. All configuration is accessible via the Mattermost UI.

Author: ```suggestion
7. All configuration is accessible via the Mattermost interface.
```

> 8. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Plugin Marketplace. It should follow [the documentation's suggested naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).

Author: ```suggestion
8. The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Marketplace. It should follow [the documentation's suggested naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).
```
